modern/src/App.js

```diff
@@ -289,7 +289,7 @@ function App() {
   React.useEffect(() => {
     getSkin().then(async root => {
       // Execute scripts
-      await Utils.asyncTreeFlatMap(root, async node => {
+      await Utils.asyncTreeFlatMap(root, node => {
```

Comment:
Guess we could write an even simpler `treeFlatMap` that doesn't have any of the `async` logic, whoops!

No need to hold up the merge for that though.